### PR TITLE
Adds support for passing list of external IPs when creating services

### PR DESCRIPTION
**Pull request by briandeheus**
**What this PR does / why we need it**:
This change will allow users to pass on a list of external IPs when creating services/exposing deployments/services. Currently the expose command only takes in a single IP, which is then mapped to `externalIPs`. This change will bring uniformity to the cli and the service definitions.

**Which issue(s) this PR fixes**:
n/a

**Additional notes**
I would to receive feedback on whether this is the right approach.

/hold
/sig cli

### CODE REVIEW

**Comment by briandeheus** [comment]
Signed the CLA!

File: pkg/kubectl/cmd/expose.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func NewCmdExposeService(f cmdutil.Factory, streams genericclioptions.IOStreams)
 	cmd.Flags().String("container-port", "", i18n.T("Synonym for --target-port"))
 	cmd.Flags().MarkDeprecated("container-port", "--container-port will be removed in the future, please use --target-port instead")
 	cmd.Flags().String("target-port", "", i18n.T("Name or number for the port on the container that the service should direct traffic to. Optional."))
-	cmd.Flags().String("external-ip", "", i18n.T("Additional external IP address (not managed by Kubernetes) to accept for the service. If this IP is routed to a node, the service can be accessed by this IP in addition to its generated service IP."))
+	cmd.Flags().String("external-ips", "", i18n.T("Additional external IP addresses (not managed by Kubernetes) to accept for the service. If these IPs are routed to a node, the service can be accessed by these IPs in addition to its generated service IP."))
```
Comment (dixudx) [inline pkg/kubectl/cmd/expose.go]: Why not setting this as `StringSlice` or `StringArray`?

**Review by dixudx — CHANGES_REQUESTED** [review]

File: pkg/kubectl/cmd/expose.go (hunk 1 above)
Comment (briandeheus) [reply pkg/kubectl/cmd/expose.go]: Good idea! I will change this.

Comment (mengqiy) [inline pkg/kubectl/cmd/expose.go]: Changing flag name is a breaking change.

**Pull request closed without merging**